parser/tests/Create.test

```diff
@@ -139,6 +139,62 @@ CREATE TABLE test (attr INT) CHECK(attr>0)
                              ^
 ==
 
+# Empty partitioning attribute is not allowed
```

Comment:
Just write the correct query, and run the test. It should fail, but the output file should have the printed statement, then copy the statement to the test file, and it should pass.